setup.py

```diff
@@ -764,7 +767,7 @@ def _picard_packages():
         'install': picard_install,
         'install_locales': picard_install_locales,
         'update_constants': picard_update_constants,
-        'get_po_files': picard_get_po_files,
+        'pull_translations': picard_pull_translations,
```

Comment:
I wonder if we should keep `get_po_files` as an alias for a while.

Reply:
There are not many users of this I think, and adapting to it is a quick change. I guess if you get an error message it is easy to update any script or whatever one might have.